src/components/annotate/modal/ValueField.tsx

```diff
@@ -22,12 +22,14 @@ const ValueField = (props: Props) => {
     const { name, property, targetType } = props;
 
     /* Base variables */
-    const annotationFilterLayer: Dict = AnnotationFilterLayer;
+    const harmonisedAttributes: Dict = HarmonisedAttributes;
     const digitalMediaFilterLayer: Dict = DigitalMediaFilterLayer;
     let filteredAnnotation;
 
     if (targetType === 'digital_specimen') {
-        filteredAnnotation = annotationFilterLayer[property];
+        console.log(property);
```

Comment:
We can probably remove the console.log right?

Reply:
Removed